deepchem/feat/molecule_featurizers/mat_featurizer.py

```diff
@@ -0,0 +1,92 @@
+from deepchem.feat.base_classes import MolecularFeaturizer
+from deepchem.utils.molecule_feature_utils import one_hot_encode
+try:
+  import logging
+  import numpy as np
+  import pandas as pd
+  import torch
+  from rdkit import Chem
+  from rdkit.Chem import AllChem
+  from rdkit.Chem import MolFromSmiles
+  from sklearn.metrics import pairwise_distances
+  from torch.utils import datasets
+except:
+  raise ImportError("Required Modules not found.")
+
+
+class MATFeaturizer(MolecularFeaturizer):
+  """
+  This class is a featurizer for the Molecule Attention Transformer.
```

Comment:
Can you expand this docstring with an explanation of what this featurizer does, an example of usage, and a link to the original paper?

Reply:
I've added the others, however I'm not sure where I should add the usage example